gr-qtgui/grc/qtgui_time_raster_x.block.yml

```diff
@@ -31,6 +31,52 @@ parameters:
     label: Num. Cols
     dtype: int
     default: 256
+<<<<<<< HEAD
```

Comment:
Looks like some merge conflicts snuck in here

Reply:
Ugh.  Git and I weren't agreeing the other day.  I thought I fixed all of those.  Just pushed an update.